arch/arm/mach-omap/am335x/mcasp_am335x.h

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ * @Component			OMAPCONF
+ * @Filename			mcasp_am335x.h
+ * @Description			Multichannel Audio Serial Port
+ * @Author			Misael Lopez Cruz (misael.lopez@ti.com)
+ * 				Adapted to AM335x by jramirez@baylibre.com
+ * @Date			2018
+ * @Copyright			Texas Instruments Incorporated
+ *
+ *
+ * Copyright (C) 2018 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *    Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *
+ *    Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the
+ *    distribution.
+ *
+ *    Neither the name of Texas Instruments Incorporated nor the names of
+ *    its contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ *  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ *  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#ifndef __MCASP_AM335X_H__
+#define __MCASP_AM335X_H__
+
+#include <stdio.h>
+
+#define MCASP1			1
+#define MCASP2			2
+
+#define MCASP1_CFG_BASEADDR	0x48038000
+#define MCASP2_CFG_BASEADDR	0x4803C000
+
+#define MCASP_PID_OFFSET	0x00
+#define PWRIDLESYSCONFIG_OFFSET	0x04
```

Comment:
Why is this one register named differently? Why not just MCASP_SYSCONFIG_OFFSET?

Reply:
That's the register name as per TRM. It's the only register whose name is not prefixed with "MCASP_".